src/App.tsx

```diff
@@ -1,61 +1,113 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const [count, setCount] = useState(0);
+  const [todos, setTodos] = useState(todosFromServer);
+  const [newTodo, setNewTodo] = useState<Todo>({
+    id: Math.max(...todos.map(todo => todo.id)) + 1,
+    title: '',
+    completed: false,
+    userId: 0,
+  });
+
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isSelectError, setIsSelectError] = useState(false);
+
+  function handleTitleChange(e: React.ChangeEvent<HTMLInputElement>) {
+    setNewTodo(currentNewTodo => ({
+      ...currentNewTodo,
+      title: e.target.value.trim(),
+    }));
+
+    setIsTitleError(false);
+  }
+
+  function handleSelectUser(e: React.ChangeEvent<HTMLSelectElement>) {
+    setNewTodo(currentNewTodo => ({
+      ...currentNewTodo,
+      userId: +e.target.value,
+    }));
+
+    setIsSelectError(false);
+  }
+
+  function handleSubmit(e: React.FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+
+    if (!newTodo.title) {
```

Comment:
```suggestion
    if (!newTodo.title.trim()) {
```
to fix it
<img width="270" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/d638eb38-9b61-45c8-81d2-9c3b36fa3318">